openpype/settings/entities/schemas/projects_schema/schema_project_deadline.json

```
@@ -289,6 +289,15 @@
                             "type": "text",
                             "key": "group",
                             "label": "Group Name"
+                        },
+                        {
+                            "type": "enum",
+                            "key": "plugin",
+                            "label": "Deadline plugin",
```

Labels differed between AYON and OpenPype - AYON has uppercase P on Plugin?
```suggestion
                            "label": "Deadline Plugin",
```